user-interface/src/main/java/life/qbic/datamanager/parser/MetadataConverter.java

```diff
@@ -0,0 +1,352 @@
+      hitMap = countHits(formattedProperties,
+          Arrays.stream(NGSMeasurementProperty.values())
+              .map(NGSMeasurementProperty::propertyName).collect(
+                  Collectors.toSet()), MEASUREMENT_ID.propertyName());
+    } else {
+      hitMap = countHits(formattedProperties,
+          Arrays.stream(NGSMeasurementProperty.values())
+              .map(NGSMeasurementProperty::propertyName).collect(
+                  Collectors.toSet()));
+    }
+    var missingProperties = new ArrayList<>();
+    for (Entry<String, Integer> entry : hitMap.entrySet()) {
+      if (entry.getValue() == 0) {
+        missingProperties.add(entry.getKey());
+      }
+    }
+    if (missingProperties.isEmpty()) {
+      return true;
+    } else {
+      log.debug("Missing properties for NGS measurement: %s".formatted(missingProperties));
+    }
+    return false;
+  }
+
+  private boolean looksLikeProteomicsMeasurement(Collection<String> properties, boolean ignoreID) {
+    var formattedProperties = properties.stream().map(String::toLowerCase)
+        .collect(Collectors.toList());
+    Map<String, Integer> hitMap;
+    if (ignoreID) {
+      formattedProperties.remove(MEASUREMENT_ID.propertyName());
+      hitMap = countHits(formattedProperties,
+          Arrays.stream(ProteomicsMeasurementProperty.values())
+              .map(ProteomicsMeasurementProperty::propertyName).collect(
+                  Collectors.toSet()), MEASUREMENT_ID.propertyName());
+    } else {
+      hitMap = countHits(formattedProperties,
+          Arrays.stream(ProteomicsMeasurementProperty.values())
+              .map(ProteomicsMeasurementProperty::propertyName).collect(
+                  Collectors.toSet()));
+    }
+    var missingProperties = new ArrayList<>();
+    for (Entry<String, Integer> entry : hitMap.entrySet()) {
+      if (entry.getValue() == 0) {
+        missingProperties.add(entry.getKey());
+      }
+    }
+    if (missingProperties.isEmpty()) {
+      return true;
+    } else {
+      log.debug("Missing properties for proteomics measurement: %s".formatted(missingProperties));
+    }
+    return false;
+  }
+
+
+  enum ProteomicsMeasurementProperty {
+    MEASUREMENT_ID("measurement id"),
+    QBIC_SAMPLE_ID("qbic sample id"),
+    SAMPLE_POOL_GROUP("sample pool group"),
+    ORGANISATION_ID("organisation id"),
+    FACILITY("facility"),
+    INSTRUMENT("instrument"),
+    CYCLE("cycle/fraction name"),
+    DIGESTION_METHOD("digestion method"),
+    DIGESTION_ENZYME("digestion enzyme"),
+    ENRICHMENT_METHOD("enrichment method"),
+    INJECTION_VOLUME("injection volume (ul)"),
+    LC_COLUMN("lc column"),
+    LCMS_METHOD("lcms method"),
+    LABELING_TYPE("labeling type"),
+    LABEL("label"),
+    COMMENT("comment");
+
+    private final String name;
+
+    ProteomicsMeasurementProperty(String value) {
+      this.name = value;
+    }
+
+    static String trimValue(String value) {
```

Comment:
This method confuses me. Why did you introduce it? why not call `trim().toLowerCase()`
instead?
It does not only trim the value but also puts it to lower case.
If you want a method to sanitize the input consider renaming the method to `prepareValue`, `sanitizeValue`, ...